resources/views/files/translate.twig

```diff
@@ -116,6 +116,9 @@ $(document).ready(function() {
         <div class="col-md-6 col-md-pull-6">
             <p>For strings that don't need translation, because they are the same as the English original, the "Needs work" checkbox should be unmarked. Otherwise the percentage display on the overview page doesn't reach 100%.</p>
         </div>
+        <div class="col-md-6 col-md-pull-6">
```

Comment:
Remove this, please.  This adds a third column and as you can see there is no place for that ;)